### PR TITLE
Add disclaimer on EVM sidechains pages 

### PR DESCRIPTION
**Pull request by amarantha-k**

### CODE REVIEW

File: content/_snippets/evmsc-disclaimer.md [hunk 1]
```diff
@@ -0,0 +1 @@
+_The EVM compatible sidechains implementation :not_enabled: is a Proof of Concept extension to the XRP Ledger protocol and is for development purposes only. There isn't an official amendment and it is not available on the production Mainnet. The EVM compatible sidechain bridge is connected to the XRP Ledger Devnet. Do not sent transactions in Mainnet._
```
Comment (DennisDawson) [inline content/_snippets/evmsc-disclaimer.md]: Does proof of concept need to be capitalized?

I don't mind this, I might even prefer it, but is it okay to use _isn't _ and _is not in close proximity this way? "There_ **isn't** an official amendment and it **is not**...."

Do not **send** transactions in Mainnet.

**Review by DennisDawson — CHANGES_REQUESTED** [review]
One definite typo, a couple of style questions.

**Review by DennisDawson — APPROVED** [review]
I fixed the typo, consider my style questions.

File: content/_snippets/evmsc-disclaimer.md (hunk 1 above)
Comment (amarantha-k) [reply content/_snippets/evmsc-disclaimer.md]: Thanks for fixing the typo. I've incorporated your style feedback.

**Pull request merged**